### ISSUE TITLE
Fatal error upon start: incompatible versions passed: from: 2.6.0, to: 2.8.2

### ISSUE THREAD

**Issue by stelgenhof** [Baldinof/roadrunner-bundle#76]
I get the following error when starting roadrunner:

```
handle_serve_command: Init error:
	endure_initialize:
	endure_internal_init: Function call error:
	endure_call_init_fn:
	config_plugin_init: incompatible versions passed: from: 2.6.0, to: 2.8.2
```

Perhaps not directly an issue with this bundle, however wanted to submit this as maybe others have experienced this as well. 

Steps to reproduce:
1. Start a new symfony project: `symfony new <name>`
2. Install this bundle: `composer require baldinof/roadrunner-bundle`
3.  Download RoadRunner locally: `vendor/bin/rr get --location bin/`
4.  Run the application: `bin/rr serve -c .rr.dev.yaml --debug`

PHP Version: `PHP 8.1.3 (cli) (built: Feb 16 2022 13:27:56) (NTS)`
OS: `Linux 5.16.12-arch1-1 #1 SMP PREEMPT Wed, 02 Mar 2022 12:22:51 +0000 x86_64 GNU/Linux`

**Comment by Baldinof** [comment]
Hi!

I think the config file created by the recipe is not up to date with the last RR version.

Can you try to change

``` yaml
server:
  command: "php public/index.php"
  env:
    APP_RUNTIME: Baldinof\RoadRunnerBundle\Runtime\Runtime
```

to

```yaml
version: "2.7"
server:
  command: "php public/index.php"
  env:
    - APP_RUNTIME: Baldinof\RoadRunnerBundle\Runtime\Runtime
```

**Comment by Baldinof** [comment]
It should be fixed with `v2.2.1`, can you try it?

**Comment by stelgenhof** [comment]
Thanks! After fixing the config it worked. Also I had to correct my `.env` as the syntax of the provided syntax regarding the `env` section is not correct. I see that `v2.2.1` has fixed that as well.

All runs fine now. Thanks for the quick fix!
